lib/repo.ts

```diff
@@ -16,7 +16,23 @@ export interface IRepo {
 }
 
 export interface BuildSourceOptions {
+  /**
+   * Single branch
+   *
+   * Cannot be specified together with `branches`.
+   *
+   * @default - All branches
+   */
   branch?: string;
+
+  /**
+   * Multiple branches
+   *
+   * Cannot be specified together with `branch`.
```

Comment:
why not just concat `[ ...(branch ? [branch] : []), ...branches ?? [] ]`?

Reply:
Fair point. Should have deprecated `branch`.